pyrpl/software_modules/lockbox/models/fabryperot.py

```diff
@@ -104,18 +83,19 @@ def calibrate(self, autosave=False):
         else:
             return None
 
-    def fit_with_voigt(self, curve, voltage):
+    def fit(self, curve, voltage):
```

Comment:
Why change the name? Perhaps other fits, e.g. including thermal effects, will be necessary.

Reply:
I renamed it to make this implementation an example of a general implementation of an input with a fit, where the details hidden behind the "fit" method would encode the type of fit, to be reflected in the class name later on. So if we had a second fit type, I would rather go with a class `VoigtFitFPInput` with a method `fit`, than a `FitFPInput` class with method `fit_with_voigt`.

Reply:
and with the way its written here, we can simply inherit from any Fit Input class, and replace the fit method to have a different fit implemented.